src/app/create-event/page.tsx

```diff
@@ -95,11 +92,11 @@ export default function CreateEvent() {
         console.error('Unexpected error:', error)
       }
     }
-
-    // Call handleSaveResponse to save the response
-    handleSaveResponse()
+    // handleSaveResponse() // Call handleSaveResponse to save the response
     setIsAvailable(false) // Set availability to false when user creates event/saves their availability
-    setIsButtonsVisible(false) // Hide buttons after creating event
+    setIsButtonsVisible(false) // Hide buttons after user creates event
+
+    // print earliest time from event
```

Comment:
missing something here?

Reply:
This was a debug comment I forgot to remove. Removing this in next commit, thanks for the catch!